.github/ci-files/prepare_mautic.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+apt-get update && apt-get install default-mysql-client -y
+
+cd /var/www
+
+cat << 'EOF' > docroot/app/config/parameters_local.php
+<?php
+/**
+ * Parameter overrides for GitHub Actions.
+ */
+$parameters = [
+    'api_enabled'           => true,
+    'api_enable_basic_auth' => true,
+];
+EOF
+
+cat << 'EOF2' > docroot/app/config/security_local.php
+<?php
+
+$this->import('security.php');
+
+// Support HTTP basic auth for test logins
+$container->loadFromExtension('security',
+    [
+        'firewalls' => [
+            'main' => [
+                // Support HTTP basic auth for test logins
+                'http_basic' => true,
+            ],
+        ],
+        'encoders'  => [
+          'Symfony\Component\Security\Core\User\User' => [
+            'algorithm'        => 'md5',
+            'encode_as_base64' => false,
+            'iterations'       => 0,
+          ],
+          'Mautic\UserBundle\Entity\User' => [
+            'algorithm'        => 'md5',
+            'encode_as_base64' => false,
+            'iterations'       => 0,
+          ],
+        ],
+    ]
+);
+EOF2
+
+php bin/console cache:clear --no-warmup
+
+php bin/console mautic:install http://localhost/ --force --mailer_from_name="GitHub Actions" --mailer_from_email="github-actions@mautic.org" --mailer_transport="smtp" --mailer_host="mailhog" --mailer_port="1025" --admin_username=admin --admin_password=mautic --admin_email="bla@bla.be"
```

Comment:
@mollux can you please remove the email values from this command as it will now fail due to https://github.com/mautic/mautic/pull/12216

And set them in the config file directly.